src/oge/emissions.py

```diff
@@ -1375,9 +1377,11 @@ def calculate_so2_from_fuel_consumption(gen_fuel_allocated, year):
         validate="m:1",
     )
     # assume all other generators have uncontrolled emissions
-    gen_fuel_allocated["so2_removal_efficiency_annual"] = gen_fuel_allocated[
-        "so2_removal_efficiency_annual"
-    ].fillna(0)
+    gen_fuel_allocated["so2_removal_efficiency_annual"] = (
+        gen_fuel_allocated["so2_removal_efficiency_annual"]
+        .astype(get_dtypes()["so2_removal_efficiency_annual"])
```

Comment:
What issue was this addressing?

Reply:
The `FutureWarning` we previously encountered: `FutureWarning: Downcasting object dtype arrays on .fillna, .ffill, .bfill is deprecated and will change in a future version`